scheduler-worker/scheduling-dsl-compiler/src/libs/scheduler-edsl-fluent-api.ts

```diff
@@ -393,7 +393,16 @@ export class Goal {
 
     let alias: string;
 
-    if (opts.forEach instanceof WindowsEDSL.Windows) {
+    let localForEach;
+    if(opts.forEach instanceof WindowsEDSL.Interval){
+      localForEach = Windows.Value(true, opts.forEach).assignGaps(false);
+    } else if(opts.forEach instanceof Temporal.Instant){
+      localForEach = Windows.Value(true,Interval.At(opts.forEach)).assignGaps(false);
+    }else{
+      localForEach = opts.forEach;
+    }
```

Comment:
Here are the spaces you and I discussed adding - it doesn't look to me like they made it into the final version

```suggestion
    if (opts.forEach instanceof WindowsEDSL.Interval) {
      localForEach = Windows.Value(true, opts.forEach).assignGaps(false);
    } else if (opts.forEach instanceof Temporal.Instant) {
      localForEach = Windows.Value(true,Interval.At(opts.forEach)).assignGaps(false);
    } else {
      localForEach = opts.forEach;
    }
```

Reply:
Updated code with spacing included